forage-android/src/main/java/com/joinforage/forage/android/vault/BasisTheoryPinSubmitter.kt

```diff
@@ -18,6 +18,8 @@ import com.joinforage.forage.android.ui.ForagePINEditText
 
 internal typealias BasisTheoryResponse = Result<Any?>
 
+internal data class ProxyRequestObject(val pin: TextElement, val card_number_token: String)
```

Comment:
We need to update the `consumer-rules.pro` file to reflect that new location of this object. Can you also add a comment about how important it is that we remember to do that in the future?

This is a very unsustainable pattern we have going here haha maybe we move this declaration to its own file with comments so we don't have to worry about it again?

Reply:
Good call!

Handled here: b8a15f15c6332da2479bc740f48ddf46dfbcc4f0